ui/helpers/taskcluster.js

```diff
@@ -136,4 +137,59 @@ export const getAction = (actionArray, actionName) => {
   return action;
 };
 
+// Bug 1638424 - Transform WPT test paths to look like paths
+// from a local checkout
+export const transformTestPath = (path) => {
+  let newPath = path;
+  // WPT path transformations
+  if (path.startsWith('/_mozilla')) {
+    // /_mozilla/<path> => testing/web-platform/mozilla/tests/<path>
+    const modifiedPath = path.replace('/_mozilla', '');
+    newPath = `testing/web-platform/mozilla/tests${modifiedPath}`;
+  } else if (path.startsWith('/')) {
+    // /<path> => testing/web-platform/tests/<path>
+    newPath = `testing/web-platform/tests${path}`;
+  }
+
+  return newPath;
+};
+
+export const transformedPaths = (manifestsByTask) => {
+  const newManifestsByTask = {};
+  Object.keys(manifestsByTask).forEach((taskName) => {
+    newManifestsByTask[taskName] = manifestsByTask[taskName].map((testPath) =>
+      transformTestPath(testPath),
+    );
+  });
+  return newManifestsByTask;
+};
+
+export const fetchGeckoDecisionArtifact = async (
+  project,
+  revision,
+  filePath,
+) => {
+  let artifactContents = {};
+  const rootUrl = prodFirefoxRootUrl;
+  const url = `${checkRootUrl(
+    rootUrl,
+  )}/api/index/v1/task/gecko.v2.${project}.revision.${revision}.taskgraph.decision/artifacts/public/${filePath}`;
+  const response = await fetch(url);
+  console.log(response.headers.get('Content-Type'));
```

Comment:
Remove the console.log

Reply:
I'm not seeing any error message being returned. Are you planning to use something generic?